extensions/amazon-sales/index.md

```diff
@@ -23,7 +23,17 @@ Amazon Sales Channel install is a `.zip` file available from the Magento Marketp
 1. Click **Download**.
 1. Export the contents to your Magento root.
 1. Follow all instructions for Extension [Installation]({{site.baseurl}}/extensions/install/).
-1. After installing, enter an API Key to complete configuration.
+1. Update your `composer.json` file
+    - Add the extension’s name and version to your composer.json file:
+    - Navigate to your Magento project directory and update your composer.json file.
+
+      ```bash
+      composer require magento/services-connector:~1.0.3
```

Comment:
Is this correct? Shouldn't it be `magento/amazon-sales-channel`?

Reply:
apparently there is an issue with the service connector.   the engineer gave this info
composer update – will update all components including their dependencies
composer require magento/services-connector:~1.0.3 – will install connector only

Reply:
The instructions are for installing the extension though. So would they need to update their composer for `magento/amazon-sales-channel` and `magento/services-connector:~1.0.3` then?

Reply:
if they don't do this, they are having service connector errors.  i am unfamiliar with the install process.  when i asked if we needed to update the cli installation, i was told "they aren't installing it that way. we just need to add this step".

Reply:
i can add to the step language, update the composer for service connector.